mjpc/planners/ilqg/planner.cc

```diff
@@ -322,8 +355,10 @@ void iLQGPlanner::GUI(mjUI& ui) {
       {mjITEM_SLIDERINT, "Rollouts", 2, &num_trajectory, "0 1"},
       // {mjITEM_RADIO, "Action Lmt.", 2, &settings.action_limits, "Off\nOn"},
       // {mjITEM_SLIDERINT, "Iterations", 2, &settings.max_rollout, "1 25"},
-      // {mjITEM_SELECT, "Policy Interp.", 2, &policy.representation,
-      //  "Zero\nLinear\nCubic"},
+      {mjITEM_SELECT, "Policy Interp.", 2, &policy.representation,
+       "Zero\nLinear\nCubic"},
+      {mjITEM_SELECT, "Reg. Type", 2, &settings.regularization_type,
+       "Control\nState-Control\nCost-To-Go\nNone"},
```

Comment:
I think in terms of GUI names I'd go with
Control\nFeedback\nValue
I can explain offline why I call regType 1 "feedback"

Reply:
Done.